### PR TITLE
Reference VisualTest.DrawInsideOutside in doc

### PR DESCRIPTION
**Pull request by josephmure**
Fixes an oversight on my part in PR #2759.

### CODE REVIEW

File: python/doc/user_manual/statistics_on_sample.rst [hunk 1]
```diff
@@ -237,6 +237,7 @@ Graphical tests
     VisualTest.DrawUpperExtremalDependenceFunction
     VisualTest.DrawLowerTailDependenceFunction
     VisualTest.DrawLowerExtremalDependenceFunction
+    VisualTest.DrawInsideOutside
```
Comment (jschueller) [inline python/doc/user_manual/statistics_on_sample.rst:240]: pyplost/DrawInsideOutside.py is missing

Comment (jschueller) [reply python/doc/user_manual/statistics_on_sample.rst:240]: hello @josephmure do you think you can take of this for the release ?

Comment (jschueller) [reply python/doc/user_manual/statistics_on_sample.rst:240]: can you try to rebase ?

File: python/src/VisualTest_doc.i.in [hunk 2]
```diff
@@ -741,9 +741,11 @@ Examples
...
 >>> from openturns.viewer import View
->>> domain = ot.Interval([-2.0, 1.0, -1], [2., 3., 1.])
->>> U = ot.Uniform(-3, 4)
->>> dist = ot.JointDistribution([U, U, U])
+>>> domain = ot.Interval([-2.0, -1.0, 1.0], [2.0, 1.0, 3.0])
+>>> domain.setName("Cuboid")
+>>> uniform = ot.Uniform(-3.0, 4.0)
+>>> dist = ot.JointDistribution([uniform] * 3)
 >>> sample = dist.getSample(200)
+>>> sample.setName("Uniform")
```
Comment (jschueller) [inline python/src/VisualTest_doc.i.in:749]: you cannot use double quotes here as its used for docstring delimiter

Comment (josephmure) [reply python/src/VisualTest_doc.i.in]: oh, right, I forgot about that. Fixed.

File: python/doc/user_manual/statistics_on_sample.rst (hunk 1 above)
Comment (josephmure) [reply python/doc/user_manual/statistics_on_sample.rst:240]: done

File: python/src/VisualTest_doc.i.in
Comment (adutfoy) [inline python/src/VisualTest_doc.i.in:1]: The result is not nice because of the tilte which is truncated!
DrawInsideOutside

**Review by adutfoy — CHANGES_REQUESTED** [review]
Nice method! Just one concern about the title of the pyplot in VisualTest....
General comment: the list of the methods proposed by VisualTest in the "API/Statistics on sample " section is not sorted in the alphabetic order....

File: python/src/VisualTest_doc.i.in
Comment (josephmure) [reply python/src/VisualTest_doc.i.in:1]: I agree, but I don't know how to fix that. The title appears in full if you produce the plot in a script, so this is a sphinx issue.

**Comment by josephmure** [comment]
@adutfoy It seems to me the order was deliberately chosen in order to group together plots which are related (for example, the CDFPlot with the PPplot and the QQplot). I am not sure using the alphabetical order is the right call.

File: python/src/VisualTest_doc.i.in
Comment (josephmure) [reply python/src/VisualTest_doc.i.in:1]: I could not solve the sphinx issue, so I shortened the title.

Comment (josephmure) [reply python/src/VisualTest_doc.i.in:1]: See here: https://output.circle-artifacts.com/output/job/736a7b88-8313-42e9-91ac-ce278151a530/artifacts/0/html/user_manual/_generated/openturns.VisualTest.DrawInsideOutside.html

**Comment by adutfoy** [comment]
I think the alphabetic order is easier for the User to find a method. The User is not always aware of the relations between methods...Moreover, this order is the one adopted for the other sections.

**Comment by josephmure** [comment]
If we decide to reorder, it can be done in a different PR, but this one did not aim to change the organization of API doc index pages.

**Comment by jschueller** [comment]
I dont know, we also want the 4 tail functions kinda grouped, so I'd just move DrawInsideOutside just after drawhenry so it stays pseudo alphabetic

File: python/doc/user_manual/statistics_on_sample.rst [hunk 3]
```diff
@@ -222,21 +222,23 @@ Graphical tests
...
+    VisualTest.DrawParallelCoordinates
     VisualTest.DrawPPplot
     VisualTest.DrawQQplot
-    VisualTest.DrawCDFplot
     VisualTest.DrawUpperTailDependenceFunction
     VisualTest.DrawUpperExtremalDependenceFunction
-    VisualTest.DrawLowerTailDependenceFunction
-    VisualTest.DrawLowerExtremalDependenceFunction
+
+    VisualTest.DrawInsideOutside
```
Comment (jschueller) [inline python/doc/user_manual/statistics_on_sample.rst:241]: you forgot this one at the end

Comment (josephmure) [reply python/doc/user_manual/statistics_on_sample.rst]: indeed, I just fixed this

**Comment by josephmure** [comment]
After discussing the issue of the order in the doc with @adutfoy I decided to use the alphabetical order in order to restore consistency with the rest of the doc index.

**Pull request merged**